src/transformState.js

```diff
@@ -5,7 +5,32 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
```

Comment:
use destructure assignment 
```suggestion
  for (const { type, extraData } of actions) {
```